Objective C/MoPub Mediation/OptInVideoExample/OptInVideoExample/ViewController.m

```diff
@@ -18,75 +18,87 @@ @interface ViewController () <MPRewardedVideoDelegate>
 
 @implementation ViewController
 
+
+- (void) addStatus: (NSString*)string {
```

Comment:
suggestion (no-blocking) : you can make a textview if you want to have several line of logs